### DISCUSSION TITLE
SmartAutoClicker-2.1.0

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#261] [Announcements]
# Bug fixes
* The delete button for the intent actions was never shown, preventing the user to delete such action. It is now displayed when needed. https://github.com/Nain57/Smart-AutoClicker/issues/253
* When copying an event containing a toggle action, the application was messing with the new event and would cause crashes. It is now fixed with a new warning flow. https://github.com/Nain57/Smart-AutoClicker/issues/247
* When using the debug overlay, the window was keeping its playing size once the detection was stopped. This was preventing the user to move the floating menu to the extreme right of the screen. This is now fixed. https://github.com/Nain57/Smart-AutoClicker/issues/258
* The keyboard behaviour was complicated, with not a lot of ways to close it. Improvements have been made for the user experience. https://github.com/Nain57/Smart-AutoClicker/issues/242

# PlayStore version
Starting with this release (hopefully), Smart AutoClicker will be available on the Google Play Store, 

## Limitations
This version will be a bit different to the FDroid/GitHub version, it will be limited:
* Maximum detection duration of 20 minutes
* No Import/Export
* Maximum of 2 scenarios, 10 events/scenario, 5 actions/event and 2 conditions/event
* No Intent and Toggle Event actions
* No condition detection threshold
* No event state control
* No detection quality/speed control
* No end conditions
* No anti-detection

## How to remove those limitations ?
You can buy the pro version of Smart Autoclicker using the Play Store in app purchase flow. The price will vary depending on your region, in France, it is 4.99€.

## Why is this version limited ?
I am working on Smart AutoClicker since the 2020 lockdown, and as the project grows bigger, it takes more and more time for a single developer to take care of it and add more features. Being able to get a little bit of money from this project will help me getting motivated to keep it alive. So if you like the project and want to support, you know what to do !

## What will happen to the FDroid and Github version ?
Those version will keep being updated, and will be similar to the Smart Autoclicker Pro experience, no limitations. Applications will always be hacked and provided on some shady third party website, usually filled with spywares. If a user wants to a find a free version of an app, he will, so I rather provide a safe one myself.

## Any other changes ?
The PlayStore version also includes Firebase Crashlytics, which will provide me all information about Smart Autoclicker crashes and will help me fix them. All information are anonymous and nothing is collected.

## When will it be available ?
Once Google Play Store approve the application, so really soon I hope

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.1.0'>SmartAutoClicker-2.1.0</a>.</em>

**Comment by Recress** [comment]
Hello, just wanted to drop by and say thanks for this amazing tool you have developed! So far I didn't find any bugs aside from very rare app crash.

I would like to know if we have some tutorial online on how to use the Intent function properly? For example, if the app stops responding, I would like to close it an open a new instance.and
